playkitvr/src/main/java/com/kaltura/playkitvr/DefaultVRPlayerWrapper.java

```diff
@@ -68,10 +74,9 @@ public void onNotSupport(int mode) {
                 .listenGesture(new MDVRLibrary.IGestureListener() {
                     @Override
                     public void onClick(MotionEvent e) {
-                        if (player == null || player.getView() == null) {
-                            return;
+                        if (vrController != null && player != null) {
```

Comment:
Better extract player.getView() as variable.

Reply:
nothing to do